lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -365,7 +370,7 @@ def test_http_exception(
             pytest.skip("skipping " + self.test_http_exception.__name__)
 
         if self.use_index_from_metadata:
-            index = self.build_index_from_metadata()
+            index = self.build_index_from_metadata(Commons.RESULT_DATA_SAMPLE)
```

Comment:
You can reference this as `self.RESULT_DATA_SAMPLE` everywhere, regardless of whether you're in the base class or a subclass context. I also have a tendency to reference class symbols using a class name; but as Webb pointed out in a recent review, using `self` means the reference is a bit more flexible across refactoring. (At least, when you know that the current class is derived from the class defining the symbol!)